connectors/bigtable/flink-connector-gcp-bigtable/src/test/java/com/google/flink/connector/gcp/bigtable/serializers/GenericRecordTest.java

```diff
@@ -40,17 +44,73 @@
  */
 public class GenericRecordTest {
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void testCorrectSerializerInitialization(Boolean useNestedRowsMode) {
+        GenericRecordToRowMutationSerializer serializer = createTestSerializer(useNestedRowsMode);
+
+        assertEquals(TestingUtils.ROW_KEY_FIELD, serializer.getRowKeyField());
```

Comment:
nit: usually if you are going to assert against something, you want the configuration that makes the assertion true or false to be in the test method. so in this scenario, it'd be ideal if the serializer was built in the actual test method instead of in a helper function. usually i prefer more code in the test methods if that means it's easier to see what's happening. specifically, i like the arrange-act-assert model. ref: https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/

i like this pattern as a good rule of thumb: https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/